osx/README.md

```diff
@@ -14,53 +14,21 @@ You need XCode commandline tools. Check by running `gcc` in the terminal. You wi
 
 Get wxPython 3+. `brew install wxpython`
 
-### cython-hidapi
-
-Installing hidapi through pip only creates a 64-bit binary on my system, so when it is stripped to 32-bit for Plover, you cannot use the module. The workaround for this is to install `hidapi` from another source. I've forked `cython-hidapi` for this purpose. Please [build and install the fork](https://github.com/morinted/cython-hidapi).
-
 ### Pip
 
 Pip dependencies can be grabbed with:
 
-`pip install appdirs pyserial simplejson py2app Cython mock pyobjc-core pyobjc-framework-cocoa pyobjc-framework-quartz`
+`pip install appdirs pyserial simplejson py2app Cython mock hidapi pyobjc-core pyobjc-framework-cocoa pyobjc-framework-quartz`
```

Comment:
I didn't have hidapi in my requirements.txt in https://github.com/openstenoproject/plover/pull/342. I probably didn't run into it as an issue since I don't have any non-keyboard steno devices around.